mmv1/products/alloydb/Instance.yaml

```diff
@@ -61,6 +69,19 @@ examples:
       - 'reconciling'
       - 'update_time'
     skip_test: true
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'alloydb_secondary_instance_basic'
+    primary_resource_id: 'secondary'
+    vars:
+      alloydb_primary_cluster_name: 'alloydb-primary-cluster'
+      alloydb_primary_instance_name: 'alloydb-primary-instance'
+      alloydb_secondary_cluster_name: 'alloydb-secondary-cluster'
+      alloydb_secondary_instance_name: 'alloydb-secondary-instance'
+      network_name: 'alloydb-secondary-network'
+    ignore_read_extra:
+      - 'reconciling'
+      - 'update_time'
+    skip_test: true
```

Comment:
Note: Skipping to showcase creating the service networking connection while testing one where we bootstrap it.